testing/sdk_security.py

```diff
@@ -129,17 +129,19 @@ def get_permissions(service_account_name: str, role: str, linux_user: str) -> Li
         }]
 
 
-def grant_permissions(linux_user: str, role_name: str, service_account_name: str) -> None:
+def grant_permissions(linux_user: str, role_name: str, service_account_name: str, additional_permissions: List[dict]) -> None:
     log.info("Granting permissions to {account}".format(account=service_account_name))
-    permissions = get_permissions(service_account_name, role_name, linux_user)
+    default_permissions = get_permissions(service_account_name, role_name, linux_user)
```

Comment:
What if one needs to remove some of the default permissions? Would we not *only* have permissions as an input parameter with `default_permissions` used if these are `None`?

Reply:
I will change the code to pass the permissions or use the defaulted ones, in case the passed list is empty